Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -589,6 +590,15 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
                 // Interpolate to get the appropriate charge density in space
                 Real rho_val = Interp(rho, nodal, Er_stag, coarsen, i, j, 0, 0);
 
+                // Interpolate current to appropriate staggering to match E field
+                Real jtot_val = 0.;
+                if (include_resistivity_term && resistivity_has_J_dependence) {
+                    Real jr_val = Interp(Jr, Jr_stag, Er_stag, coarsen, i, j, 0, 0);
+                    Real jt_val = Interp(Jt, Jt_stag, Er_stag, coarsen, i, j, 0, 0);
+                    Real jz_val = Interp(Jz, Jz_stag, Er_stag, coarsen, i, j, 0, 0);
+                    jtot_val = sqrt(pow(jr_val, 2) + pow(jt_val, 2) + pow(jz_val,2));
```

Comment:
I think the preferred way to calculate it would be:
```suggestion
                    jtot_val = sqrt(jr_val*jr_val + jt_val*jt_val + jz_val*jz_val);
```

Reply:
I have dropped using the pow call.  Just curious though, with compiler optimization wouldn't pow(,2) be inlined anyway?

Reply:
Yes, with new enough compilers it should be but the reasoning is that with a simple square, the `x*x` approach will never be less efficient than `pow(x, 2)` whereas the latter could be less efficient with sub-optimal compiler flags. Also, I think the Clang-tidy complaint about implicit type conversion comes from the `pow` usage.

Reply:
So, actually with the change I don't think you need the `static_cast<Real>` anymore.